plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/BaseProjectTest.java

```diff
@@ -30,7 +30,7 @@
 /**
  * Common infrastructure for workbench-based tests that create a single project.
  */
-public class AbstractProjectTest {
+public class BaseProjectTest {
```

Comment:
There's no abstract method in this class, so I'll keep it `BaseProjectTest`. (It's not my code, so I don't immediately see how I could convert this to an abstract class.)